Client/src/Components/NavigationMenu.tsx

```diff
@@ -31,6 +35,15 @@ export default function NavigationMenu() {
   const handleSignIn = () => {
   }
 
+  const history = useHistory();
+
+  function logout() {
+
+    sessionStorage.clear();
+    window.location.replace("/");
```

Comment:
does this redirect to login?

Reply:
@ds161833 no, i thought it should redirect to home page instead...it redirects to homepage

Reply:
alright its okay